File: com/pdfjet/TextAnnotation.java
/*
 * TextAnnotation.java
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
package com.pdfjet;

/**
 * A text note annotation.
 */
public class TextAnnotation extends BaseAnnotation {
    /**
     * Creates a text note annotation.
     */
    public TextAnnotation() {
        super.annotationType = Annotation.Text;
    }
}

File: net/pdfjet/TextAnnotation.cs
/*
 * TextAnnotation.cs
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
using System;

namespace PDFjet.NET {
/// <summary>A text note annotation.</summary>
public class TextAnnotation : BaseAnnotation {
    /// <summary>Creates a text note annotation.</summary>
    public TextAnnotation() {
        base.annotationType = Annotation.Text;
    }
}
}
